mycroft/util/parse.py

```diff
@@ -164,6 +169,571 @@ def extractnumber_en(text):
     return val
 
 
+'''authors: spenrod and connorpenrod
+this function takes a sentence and extracts date and time information from it
```

Comment:
Since this is the primary documentation (docstrings are used to auto-generate the docs), this is probably a good place to explain some of the non-obvious behaviors this uses.  For example, an unqualified "evening" gets returned as 7pm, "morning" as 8am, "afternoon" as 3pm.